ChangeLog.md

```diff
@@ -14,6 +14,7 @@
 
 - Add default value consistently for parameters #1164
 - Make `content_type` param keyword-only if there are multiple content types #1167
+- Add default_optional_constants_to_none option for optional constant parameters #1171
```

Comment:
nit: should be included as `--default-optional-constants-to-none` since that's the flag users will use